src/stylesheets/components/_site-search.scss

```diff
@@ -38,21 +38,25 @@ $icon-size: 40px;
   }
 
   .app-site-search__wrapper {
+    $encoded-svg-fill-colour: "%23" + str-slice(ie-hex-str(govuk-colour("dark-grey")), 4);
```

Comment:
Actually, I can't really follow what this is doing – we're padding it out to include an alpha channel but then slicing it back to the first 4 characters? If it can't be simplified, this could definitely do with a comment.

Reply:
The hash in the snippet below is Sass interpolation syntax, agreed it's very complicated, will try and simplify it but not sure I'll be able to.